refine.bat

```diff
@@ -211,28 +212,34 @@ echo REFINE_MEMORY = %REFINE_MEMORY%
 echo ----------------------- 
 ) > support.log
 
-set CLASSPATH="%REFINE_CLASSES_DIR%;%REFINE_LIB_DIR%\*"
+set LIB_CLASSPATH=('type server/target/classpath.txt')
+set CLASSPATH="%REFINE_CLASSES_DIR%;%LIB_CLASSPATH%\*"
 "%JAVA_HOME%\bin\java.exe" -cp %CLASSPATH% %OPTS% -Djava.library.path=%REFINE_LIB_DIR%/native/windows com.google.refine.Refine
 goto end
 
-:doAnt
-if not "%ANT_HOME%" == "" goto gotAntHome
-echo You must have Apache Ant installed and the ANT_HOME environment variable to point to it
+:doMvn
+if not "%MAVEN_HOME%" == "" goto gotMvnHome
+echo You must have Apache Maven installed and the MAVEN_HOME environment variable to point to it.
 echo.
 echo You can download it from
 echo.
-echo   http://ant.apache.org/
+echo   https://maven.apache.org/
+echo
+echo The environment variable MAVEN_HOME should not include the final "bin" directory, such as:
+echo
+echo   C:\Program Files (x86)\Apache\Maven
 echo.
 echo If you don't know how to set environment variables, follow the instructions at
 echo.
 echo   http://bit.ly/1c2gkR
 echo.
-:gotAntHome
-"%ANT_HOME%\bin\ant.bat" -f build.xml %ACTION%
-goto end
-
-:doAntDebug
-"%ANT_HOME%\bin\ant.bat" -f -v -diagnostics build.xml %ACTION%
+:gotMvnHome
+set MVN_ACTION=""%ACTION%""
+if ""%ACTION%"" == ""build"" set MVN_ACTION=""process-resources compile dependency:build-classpath""
+if ""%ACTION%"" == ""test"" set MVN_ACTION=""process-resources compile test""
+if ""%ACTION%"" == ""server_test"" set MVN_ACTION=""process-resources compile test -f main""
+if ""%ACTION%"" == ""extensions_test"" set MVN_ACTION=""process-resources compile test -f extensions""
+"%MAVEN_HOME%\bin\mvn.cmd" %MVN_ACTION%
```

Comment:
You need to perform a "call" such as:
```
call %MAVEN_HOME%\bin\mvn.cmd %MVN_ACTION%
```
mvn.cmd is a Windows batch file and not an executable.
In Windows, if one batch file invokes another directly, when the invoked batch file finishes, script run ends. If you want the calling batch file to continue after the invoked batch file ends, you must use the "call" command.

Reply:
Great, thanks! I don't have access to a Windows machine at the moment so it's very useful.